src/olympia/reviewers/views.py

```diff
@@ -804,6 +807,9 @@ def review(request, addon, channel=None):
     else:
         flags = []
 
+    private_whiteboard = PrivateWhiteboard.objects.get(pk=addon.pk)
+    private_wb_form = forms.PrivateWhiteboardForm(instance=private_whiteboard)
```

Comment:
Don't abbreviate the variable, `private_whiteboard_form` + a line break is fine.

Reply:
Ok, will do. It looked super cludgy with a linebreak, and I think further down in `context()` this would cause me to wrap as well. I'll double check though.